src/main/java/picard/sam/DuplicationMetrics.java

```diff
@@ -24,62 +24,84 @@
 
 package picard.sam;
 
-import htsjdk.samtools.metrics.MetricBase;
 import htsjdk.samtools.util.Histogram;
+import picard.analysis.replicates.MergeableMetricBase;
```

Comment:
if MergableMetrics is going to be used more broadly, I think we should move it to htsjdk. Or atleast out of the  replicate package...